### ISSUE TITLE
wal-g crashes when datadir is symlink.

### ISSUE THREAD

**Issue by lesovsky** [wal-g/wal-g#55]
Hi,
wal-g crashes when it's trying to push backup and datadir is symlink.
It's not a critical issue, but it seems to me it's better to return error (or walk to destination dir) instead of crash.
wal-g version 0.1.3

```
postgres@pg0:~$ ls -l /var/lib/postgresql/9.6/main
lrwxrwxrwx 1 postgres postgres 16 Nov 21 07:19 /var/lib/postgresql/9.6/main -> /data/postgresql
postgres@pg0:~$ wal-g backup-push /var/lib/postgresql/9.6/main
BUCKET: production-backups
SERVER: 
Walking ...
Starting part 1 ...

Finished writing part 1.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x89766b]

goroutine 1 [running]:
github.com/wal-g/wal-g.(*Bundle).HandleSentinel(0xc4201579e0, 0x0, 0x0)
        /home/travis/gopath/src/github.com/wal-g/wal-g/upload.go:281 +0x3b
github.com/wal-g/wal-g.HandleBackupPush(0x7ffe8fd6be04, 0x1c, 0xc42011cfc0, 0xc420188660)
        /home/travis/gopath/src/github.com/wal-g/wal-g/commands.go:555 +0x796
main.main()
        /home/travis/gopath/src/github.com/wal-g/wal-g/cmd/wal-g/main.go:107 +0x6db
postgres@pg0:~$
postgres@pg0:~$
postgres@pg0:~$ wal-g backup-push /data/postgresql/
BUCKET: production-backups
SERVER: 
Walking ...
Starting part 1 ...

PG_VERSION
backup_label.old
base
base/1
....
```

**Comment by x4m** [comment]
Hi, Alexey! Thanks for reporting that. I'll fix this.

**Comment by x4m** [comment]
Actually, there is one more important issue, which cannot be easily resolved.
Many expected precondition violations (like PostgreSQL is not running) finish with stack trace and look like a crash, that is also bad and worth fixing.

But in your case, this is crash, indeed. I've addressed this issue in 027b50aea38fd914b17327917bf651da8ff15125 , I'm going to cut a release after fixing #50, hopefully this week.

I'll close the issue as for now, but feel free to ping me with any related news.
